webserver/webserver_test.go

```
@@ -52,3 +47,33 @@ func TestWebServer(t *testing.T) {
+
+func setupTestServer(t *testing.T, addr string) *webserver.WebServer {
+	wb := webserver.NewWebServer("../www")
+	testRouteResistration(wb)
+
+	listener, err := net.Listen("tcp", addr)
+	if err != nil {
+		t.Fatal(err)
+	}
+	// we set the listener here to avoid running the tests before the server is ready
+	wb.Listener = listener
+	go wb.Run(addr)
```

golangci-lint (1.62.2)

Error return value of `wb.Run` is not checked

(errcheck)